packages/sandcastle/src/Gallery/loadFromUrl.ts

```diff
@@ -43,6 +43,20 @@ export function loadFromUrl(
 ) {
   const searchParams = new URLSearchParams(window.location.search);
 
+  const codeParam = searchParams.get("code");
+  if (codeParam) {
+    // This is a legacy support type url that was used by ion.
+    // Ideally we use the #c= param as that results in shorter urls
+    // The code query parameter is a Base64 encoded JSON string with `code` and `html` properties.
+    const json = JSON.parse(window.atob(codeParam.replaceAll(" ", "+")));
```

Comment:
Do you know what the purpose of the `.replaceAll(" ", "+"))` is? Should it cover all whitespace with `/\s/g` instead?